_benchmark/user-guide/understanding-workloads/choosing-a-workload.md

```diff
@@ -20,10 +20,76 @@ Consider the following criteria when deciding which workload would work best for
 - The data types that your cluster uses compared to the data structure of the documents contained in the workload. Each workload contains an example document so that you can compare data types, or you can view the index mappings and data types in the `index.json` file.
 - The query types most commonly used inside your cluster. The `operations/default.json` file contains information about the query types and workload operations. For a list of common operations, see [Common operations]({{site.url}}{{site.baseurl}}/benchmark/user-guide/understanding-workloads/common-operations/).
 
-## General search clusters
+## General search use cases: `nyc_taxis`
 
-For benchmarking clusters built for general search use cases, start with the [nyc_taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/nyc_taxis) workload. This workload contains data about the rides taken in yellow taxis in New York City in 2015. 
+For benchmarking clusters built for general search use cases, start with the [nyc_taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/nyc_taxis) workload. It contains the contains the following:
 
-## Log data
+- **Data type:** Ride data from yellow taxis in New York City in 2015.
+- **Cluster requirements:** Suitable for small to medium-sized clusters.
```

Comment:
Do we have definitions for what a small and medium sized cluster is? (E.g. small is 3-5 node clusters while medium is 5-20 node clusters)?

Reply:
Let me look into that one. There might be AWS guidance somewhere.

Reply:
@IanHoang: See line 19 and let me know if that definition works.